src/server/firefox/ffBrowser.ts

```diff
@@ -200,6 +200,8 @@ export class FFBrowserContext extends BrowserContext {
       promises.push(this.setOffline(this._options.offline));
     if (this._options.colorScheme)
       promises.push(this._browser._connection.send('Browser.setColorScheme', { browserContextId, colorScheme: this._options.colorScheme }));
+    if (this._options.reducedMotion)
```

Comment:
For some reason, logic here is different - we don't always send the value and rely on system default.

Reply:
It's related to the question if we want no-preference by default, I think we should to have reproducible tests across operating systems. Otherwise it will end up like on the color scheme side to follow up with defaults.